Modules/Relationships/src/Relationships.Application/Relationships/Queries/CanEstablishRelationship/CanEstablishRelationshipQueryValidator.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Backbone.BuildingBlocks.Application.FluentValidation;
+using FluentValidation;
+
+namespace Backbone.Modules.Relationships.Application.Relationships.Queries.CanEstablishRelationship;
+
+// ReSharper disable once UnusedType.Global
+public class CanEstablishRelationshipQueryValidator : AbstractValidator<CanEstablishRelationshipQuery>
+{
+    public CanEstablishRelationshipQueryValidator()
+    {
+        RuleFor(q => q.PeerAddress).DetailedNotEmpty();
```

Comment:
`RuleFor(q => q.PeerAddress).Must(IdentityAddress.IsValid);`

Reply:
done

Reply:
You can remove the rule for "DetailedNotEmpty" as it's now redundant